.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
Keep "git.ignoreLimitWarnin" disable/false. Just remore node_modules, dist and other unnecessary folder from git tracking